src/VisualStudio/VisualBasic/Impl/BasicVSResources.resx

```diff
@@ -288,4 +288,7 @@
   <data name="In_relational_binary_operators" xml:space="preserve">
     <value>In relational operators:  =   &lt;&gt;   &lt;   &gt;   &lt;=   &gt;=   Like   Is</value>
   </data>
+  <data name="VB_Coding_Conventions" xml:space="preserve">
+    <value>VB Coding Conventions</value>
```

Comment:
can we get PM signoff on using VB instead of Visual Basic?  In the past people have been sticklers about that.

Reply:
nevermind.  I see that you've mentioned this is how the docs site talks about things.